src/wrappers/express.js

```diff
@@ -75,6 +80,25 @@ function expressMiddleware(req, res, next) {
     }
 }
 
+/**
+ * Wraps express next function that calls next middleware
+ * @param {*} req express request
+ * @param {*} next express next middleware
+ * @returns {*} wrapeed function
+ */
+function nextWrapper(req, next) {
+    const originalNext = next;
+    return function internalNextWrapper(error) {
+        if (error) {
+            utils.debugLog('Epsagon Next - middleware executed');
+            utils.debugLog(error);
+        }
+        traceContext.setTraceToEpsagonId(req.epsagonId);
+        originalNext(...arguments);
+        traceContext.setTraceToEpsagonId(req.epsagonId);
```

Comment:
Why is this line needed before and after?

Reply:
The first line set trace before the middleware is running
The second line updates trace if needed